src/app/components/profile/profile.component.html

```diff
@@ -0,0 +1,31 @@
+<mat-card>
+  <mat-card-header>
+    <mat-card-title>Name</mat-card-title>
+  </mat-card-header>
+  <mat-card-content>
+    <p>
+      {{ name }}
+    </p>
+  </mat-card-content>
+
+  <mat-card-header>
+    <mat-card-title>Email</mat-card-title>
+  </mat-card-header>
+  <mat-card-content>
+    <p>
+      {{ email }}
+    </p>
+  </mat-card-content>
+</mat-card>
+
+<mat-card class="mt-3">
+  <mat-card-header>
+    <mat-card-title>Shiba Inu</mat-card-title>
+  </mat-card-header>
+  <mat-card-content>
+    <p>The Shiba Inu is the smallest of the six original and distinct spitz breeds of dog from Japan.</p>
+  </mat-card-content>
+  <mat-card-actions>
+    <button mat-raised-button color="accent" (click)="openDialog()">Delete</button>
```

Comment:
do not use the result of the action in the names, use the name of the action. (delete() or onClickDelete())

Reply:
Done